lib/views/create-bug.js

```diff
@@ -122,6 +116,7 @@ function bugChanged() {
     clearTimeout(timeout);
   }
   timeout = setTimeout(updateBugData, 1000);
```

Comment:
Yeh its just debouncing the changes since this will get called on every letter you type in a text area and would prefer to avoid thrashing indexedDB